### PR TITLE
Upgrade config for php-cs-fixer v2

### PR DESCRIPTION
**Pull request by julienbourdeau**

### CODE REVIEW

**Comment by rayrutjes** [comment]
Can we add the command as a dev dependency as well?
Maybe also add it as a composer script could help us running it more often.

**Comment by julienbourdeau** [comment]
I tried but as far as I understood, we can't because we require `php >= 5.3` and it seems incompatible. I'll see if I find a way.
At least I wanted to be able to use it locally (my test failed previously because I used the array short syntax).

**Comment by rayrutjes** [comment]
php-cs-fixer is super confusing, and the fact that we still need to support 5.3 definitely makes it trickier :/

File: .php_cs [hunk 1]
```diff
@@ -10,6 +10,10 @@ return PhpCsFixer\Config::create()
     ->setRules([
         '@Symfony' => true,
         'array_syntax' => ['syntax' => 'long'],
+        'trailing_comma_in_multiline_array' => false,
```
Comment (julienbourdeau) [inline .php_cs]: I think this was on before. Why not leaving the Symfony default?

Comment (rayrutjes) [reply .php_cs]: No it was off. When I turn it on, it adds a trailing coma everywhere.
I personally prefer it on though. I was about to ask you your feedback on the PR.

**Comment by rayrutjes** [comment]
Let me know if there are rules you'd like to amend @julienbourdeau @JanPetr

**Comment by julienbourdeau** [comment]
Rebased and removed commit merge. I'll fix the master branch after it's merged (to avoid conflicts while this PR is pending).

**Comment by julienbourdeau** [comment]
Because the v2 is around the corner, I'd rather avoid having this kind of big change on the v1 now. I'll close it, if we really need to fix the codebase, we can do it but I think we're fine. Thanks Raymond for the help.

**Pull request closed without merging**